src/physlr-molecules.cc

```diff
@@ -0,0 +1,401 @@
+#include <fstream>
+#include <getopt.h>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <boost/graph/adjacency_list.hpp>
+#include <boost/graph/biconnected_components.hpp>
+#include <boost/graph/graph_utility.hpp>
+#include <boost/graph/subgraph.hpp>
+
+#if _OPENMP
+#include <omp.h>
+#endif
+
+#define PROGRAM "physlr-molecules"
+#define PACKAGE_NAME "physlr"
+#define GIT_REVISION "pre-autotools"
+
+static int
```

Comment:
Prefer `size_t` or `unsigned` for unsigned integers.

Reply:
done